packages/data/platform/users_repository_flutter/lib/src/storage_json.dart

```diff
@@ -88,7 +94,10 @@ class UsersRepositoryStorageJson implements UsersRepository {
     /// make lastLogin equal true
     /// and return him
     final user = this.get(id);
-    if (user != null) user.lastLogin = true;
+    if (user != null) {
+      print("OK");
```

Comment:
I would revert this, OK anyway is not so informative

Reply:
Oh, it is from time I struggled with #30.
I need to change a few lines. Thank you!